mixins/arrow-keys-mixin.js

```diff
@@ -96,6 +96,7 @@ export const ArrowKeysMixin = superclass => class extends superclass {
 		} else {
 			return;
 		}
+		e.stopPropagation();
```

Comment:
Without this, the event would bubble up and the list-item's generic layout code would take control of focus when we don't want it to.  We want focus to be managed by the interactive component until they exist interactive mode.